src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+import mypackage.Color;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int indexColor = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[indexColor];
+        return String.valueOf(color);
```

Comment:
Please refer to checklist:
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants